nixos/modules/services/misc/synergy.nix

```diff
@@ -88,17 +106,29 @@ in
         description = "Synergy client";
         wantedBy = optional cfgC.autoStart "multi-user.target";
         path = [ pkgs.synergy ];
-        serviceConfig.ExecStart = ''${pkgs.synergy}/bin/synergyc -f ${optionalString (cfgC.screenName != "") "-n ${cfgC.screenName}"} ${cfgC.serverAddress}'';
+        serviceConfig.ExecStart =
+            ''${pkgs.synergy}/bin/synergyc''
+          + " --no-daemon "
+          + optionalString (cfgC.screenName != "") "-n ${cfgC.screenName} "
+          + optionalString (cfgC.enableCrypto)     "--enable-crypto"
+          + cfgC.serverAddress;
         serviceConfig.Restart = "on-failure";
       };
     })
     (mkIf cfgS.enable {
-      systemd.services."synergy-server" = {
+      systemd.user.services."synergy-server" = {
```

Comment:
This requires the user to be logged in (so that a `systemd --user` instance is spawned). Is this desired for this kind of service?

Reply:
That's how I use it, at least.  If we want to have synergy working on the display manager login screen, it would probably have to run as root.  My guess is that nearly all synergy users will find it acceptable to have synergy start after login, but I suppose I don't know that for certain.

Reply:
ok